integrations/wootric/test/index.test.js

```diff
@@ -221,7 +221,9 @@ describe('Wootric', function() {
         analytics.assert(window.wootricSettings.properties.property2 === 'bar');
       });
 
-      it('should set suffix _date to property key and convert date if trait is a date', function() {
```

Comment:
This test didn't work before and is only failing now because [we only test commit changes](https://github.com/segmentio/analytics.js-integrations/blob/0460b7f2ee674525651725ecd7f8029f7949161b/package.json#L14). We can verify this by running:
```bash
git checkout origin/master
git clean -fdx
yarn
yarn workspace @segment/analytics.js-integration-wootric run test
```